group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -1905,7 +1718,7 @@ private CoordinatorResult<ConsumerGroupHeartbeatResponseData, CoordinatorRecord>
                 metrics.record(CONSUMER_GROUP_REBALANCES_SENSOR_NAME);
             }
 
-            group.setMetadataRefreshDeadline(currentTimeMs + consumerGroupMetadataRefreshIntervalMs, groupEpoch);
+            group.setMetadataRefreshDeadline(currentTimeMs + Integer.MAX_VALUE, groupEpoch);
```

Comment:
It seems the "deadline" is a no-op due to `Integer.MAX_VALUE`, correct?

Reply:
Correct. We don't rely on periodic refreshed at the moment. Let me still extract it into a constant.